pkg/reaper/reaper.go

```diff
@@ -215,3 +252,15 @@ func maxTTL(resourceA, resourceB *resources.WatchedResource) (string, error) {
 		return resourceB.TTL, nil
 	}
 }
+
+// parseSchedule parses the cron time string that defined the reaper's
+// run schedule, and either returns a Schedule struct, or nil if the
+// schedule string is malformed.
+func parseSchedule(schedule string) cron.Schedule {
```

Comment:
Do we want to return an error here in addition to nil in the error found case?

Reply:
Function now returns schedule and error